hamilton/function_modifiers/metadata.py

```diff
@@ -271,3 +282,38 @@ def example_schema() -> pd.DataFrame:
         Then, when drawing the DAG, the schema will be displayed as sub-elements in the node for the DAG (if `display_schema` is selected).
         """
         return SchemaOutput(*fields, target_=target_)
+
+
+class RayRemote(tag):
+
+    def __init__(self, **options: Union[int, Dict[str, int]]):
+        """Initializes RayRemote. See docs for `@ray_remote_options` for more details."""
+
+        ray_tags = {f"ray_remote.{option}": json.dumps(value) for option, value in options.items()}
+
+        super(RayRemote, self).__init__(bypass_reserved_namespaces_=True, **ray_tags)
+
+
+def ray_remote_options(**kwargs: Union[int, Dict[str, int]]) -> RayRemote:
```

Comment:
can we also get a unit test on this to check that the tags are created as expected please.

You can do something similar to `test_decorate_node_with_schema_output` in `test_metadata.py`.